enginetest/queries.go

```diff
@@ -404,6 +404,10 @@ var QueryTests = []QueryTest{
 		Query:    "select * from mytable t1 join mytable t2 on t2.i = t1.i where t2.i > 10",
 		Expected: []sql.Row{},
 	},
+	{
+		Query:    "select * from mytable t1 join mytable T2 on T2.i = t1.i where T2.i > 10",
```

Comment:
Should vary the case of the alias between declaration and use

Reply:
need lower case numbers